config/config.go

```diff
@@ -335,6 +339,19 @@ func ReadFromFile(fn string) (*Struct, error) {
 	if err := json.Unmarshal(b, &cfg); err != nil {
 		return nil, fmt.Errorf("decoding %s: %v", fn, err)
 	}
+
+	if cfg.Sops != nil {
+		cmd := exec.Command("sops", "decrypt", fn)
```

Comment:
I don’t like that this integration is so coupled to sops.

If we end up changing the code, I think we should have a somewhat generic “run this command before reading the config” hook, but maybe that’s cleaner to keep outside the `gok` invocation; see also https://github.com/gokrazy/gokrazy/issues/282#issuecomment-2676050978